packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -213,6 +223,68 @@ export class SchematizingSimpleTreeView<
 		return this.view;
 	}
 
+	/**
+	 * {@inheritDoc @fluidframework/shared-tree#TreeViewAlpha.runTransaction}
```

Comment:
@noencke @yann-achard-MS I am not sure if this will find the right doc comments. Do you guys know how I can tell?

Reply:
In my experience, @inheritDoc just doesn't work half the time, no matter how much you try to help it out. Sometimes it even works in the wrong way, and does things that are unexpected. I simply avoid using it.